anon/compat.py

```diff
@@ -0,0 +1,13 @@
+# deps
+from django_bulk_update.helper import bulk_update as ext_bulk_update
+
+
+def bulk_update(objects, manager, **bulk_update_kwargs):
+    """ Updates the list of objects using django queryset's
+        inbuilt `.bulk_update()` method if present, else
+        django_bulk_update's `bulk_update` will be used
+    """
+    if getattr(manager, "bulk_update", None):
+        manager.bulk_update(objects, **bulk_update_kwargs)
+    else:
+        ext_bulk_update(objects, bulk_update_kwargs)
```

Comment:
🎨 It's easier to ask for forgiveness than permission (EAFP):

[https://docs.python.org/3.4/glossary.html](https://docs.python.org/3.4/glossary.html)

```suggestion
    try:
        manager.bulk_update(objects, **bulk_update_kwargs)
    except AttributeError:
        ext_bulk_update(objects, bulk_update_kwargs)
```

Reply:
Updated the code with your suggestion.